### PR TITLE
Edits for PR #18098

### PR DESCRIPTION
**Pull request by dharmabumstead**
##### ISSUE TYPE

- Feature Pull Request
- New Module Pull Request
- Bugfix Pull Request
- Docs Pull Request
##### ANSIBLE VERSION

```

```
##### SUMMARY

```

```

### CODE REVIEW

File: docsite/rst/intro_configuration.rst [hunk 1]
```diff
@@ -105,25 +105,24 @@ It is useful when becoming an unprivileged user::
...
+This string can be helpful to indicate that a file should not
+be directly edited because Ansible may overwrite the contents of the file.
 
-There are several special values that, when placed in the string, will be
-replaced with useful information.  These are not in the default
-``ansible_managed`` string because they can cause Ansible to think that the
-template has changed when it's only the ansible_managed string which has
-changed.  These values, along with the situations which can lead Ansible to
-report a template as changed when they are used are:
+There are several special placeholder values that be placed in the ``ansible_managed`` string to act as macros.  These are not in the default ``ansible_managed`` string because they can cause Ansible to behave as though the
```
Comment (abadger) [inline docsite/rst/intro_configuration.rst:120]: Maybe take out the macros portion.

> There are several special placeholder values that be placed in the `ansible_managed` string.

File: docsite/rst/intro_configuration.rst [hunk 2]
```diff
@@ -105,25 +105,24 @@ It is useful when becoming an unprivileged user::
...
+entire template has changed when only the ansible_managed string which has
+changed.
```
Comment (abadger) [inline docsite/rst/intro_configuration.rst:122]: Either take out which or add it's:

> entire template has changed when only the ansible_managed string has changed.

or 

> entire template has changed when it's only the ansible_managed string which has changed.

File: docsite/rst/intro_configuration.rst [hunk 3]
```diff
@@ -1159,3 +1157,5 @@ ignore_certs
 
 If set to *yes*, ansible-galaxy will not validate TLS certificates. Handy for testing against a server with a self-signed certificate
 .
+ver with a self-signed certificate
+.
```
Comment (abadger) [inline docsite/rst/intro_configuration.rst:1161]: This looks like a mistaken cut and paste?

Comment (dharmabumstead) [reply docsite/rst/intro_configuration.rst:1161]: Don't know who wrote that; I haven't edited to that part of the file yet.

**Pull request merged**